src/custom/state/orders/updaters/PendingOrdersUpdater.ts

```diff
@@ -25,10 +25,9 @@ import { OrderID } from 'api/gnosisProtocol'
 
 import { fetchOrderPopupData, OrderLogPopupMixData } from 'state/orders/updaters/utils'
 import { GetSafeInfo, useGetSafeInfo } from 'hooks/useGetSafeInfo'
-import ms from 'ms.macro'
 import { openNpsAppziSometimes } from 'utils/appzi'
 
-const PENDING_TOO_LONG_TIME = ms`5 min`
+const PENDING_TOO_LONG_TIME = 10 * 1000 // 10 seconds
```

Comment:
Nitpick

```suggestion
const PENDING_TOO_LONG_TIME = ms`10s`
```

😆

Reply:
ahahah, i will never merge this PR

in my defense, i went to their readme and didn't find anything related to seconds, so i said NVM!